policy_set_integration_test.go

```diff
@@ -165,15 +165,14 @@ func TestPolicySetsCreate(t *testing.T) {
 		assert.Equal(t, ps.Description, "")
 		assert.False(t, ps.Global)
 		assert.Equal(t, ps.PoliciesPath, "/policy-sets/foo")
-		assert.Equal(t, ps.VCSRepo, &VCSRepo{
-			Branch:            "policies",
-			DisplayIdentifier: githubIdentifier,
-			Identifier:        githubIdentifier,
-			OAuthTokenID:      oc.ID,
-			IngressSubmodules: true,
-			RepositoryHTTPURL: fmt.Sprintf("https://github.com/%s", githubIdentifier),
-			ServiceProvider:   string(ServiceProviderGithub),
-		})
+		assert.Equal(t, ps.VCSRepo.Branch, "policies")
+		assert.Equal(t, ps.VCSRepo.DisplayIdentifier, githubIdentifier)
+		assert.Equal(t, ps.VCSRepo.Identifier, githubIdentifier)
+		assert.Equal(t, ps.VCSRepo.IngressSubmodules, true)
+		assert.Equal(t, ps.VCSRepo.OAuthTokenID, oc.ID)
+		assert.Equal(t, ps.VCSRepo.RepositoryHTTPURL, fmt.Sprintf("https://github.com/%s", githubIdentifier))
+		assert.Equal(t, ps.VCSRepo.ServiceProvider, string(ServiceProviderGithub))
+		assert.Regexp(t, "^https://app\\.terraform\\.io/webhooks/vcs/[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}$", ps.VCSRepo.WebhookURL)
```

Comment:
The only thing I might change here is the host, given that someone can test against a Terraform Enterprise installation where the host might not necessarily match with `app...terraform...io`. The best choice here would be to stick the value of `TFE_ADDRESS` into the regexp.

Reply:
Yeah I see the problem. Is the rest of the URL for the webhook standard or can the endpoint and/or the UUID part also change on TFE?

Reply:
I couldn't find a way to easily access the Config object used while creating the test client, so I called DefaultConfig() myself for its Address field. DefaultConfig() function handles all cases with `TFE_ADDRESS` and `TFE_HOSTNAME` env variables (or lack thereof) so I thought it would be best to use it instead of just `TFE_ADDRESS`. If there is a better/cleaner way, feel free to point me in the right direction